ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -103,6 +103,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8
+- as: e2e-aws-ipi-disconnected-private-sno-baselinecaps-none-p2
+  cron: 0 2 */7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @disconnected and @network-ovnkubernetes and @singlenode
```

Comment:
can we exclude console cases since no console will be deployed in the workflow `cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-baselinecaps-none` ?

Reply:
**I need the exact filter for specific teams/features from you guys, or feel free to use follow up PRs to add extra filters to exclude specific tests.**

Reply:
@yapei console cases excluded in https://github.com/openshift/release/pull/33046/commits/d5301f23016f52ad7cd075c21a1cbac358d852c4